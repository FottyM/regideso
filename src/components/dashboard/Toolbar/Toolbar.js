import React from 'react'
import Share2 from 'react-feather/dist/icons/share-2'
import ExternalLink from 'react-feather/dist/icons/external-link'
import Calendar from 'react-feather/dist/icons/calendar'

const Toolbar = () => {
  return (
    <div className="d-flex justify-content-between flex-wrap flex-md-nowrap align-items-center pb-2 mb-3 border-bottom">
      <h1 className="h2">Dashboard</h1>
      <div className="btn-toolbar mb-2 mb-md-0">
        <div className="btn-group mr-2">
          <button className="btn btn-sm btn-outline-secondary">
            <Share2 className="feather" />
            Share
          </button>
          <button className="btn btn-sm btn-outline-secondary">
            <ExternalLink className="feather" />
            Export
          </button>
        </div>
        <button className="btn btn-sm btn-outline-secondary dropdown-toggle">
          <Calendar className="feather" />
          This week
        </button>
      </div>
    </div>
  )
}

export default Toolbar
