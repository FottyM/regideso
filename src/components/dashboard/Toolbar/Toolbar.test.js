import React from 'react'
import { shallow } from 'enzyme'

import Toolbar from './Toolbar'
describe('<Toolbar />', () => {
  it('renders', () => {
    const component = shallow(<Toolbar />)
    expect(component).toMatchSnapshot()
  })
})
