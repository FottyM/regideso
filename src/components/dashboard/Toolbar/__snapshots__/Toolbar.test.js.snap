// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Toolbar /> renders 1`] = `
<div
  className="d-flex justify-content-between flex-wrap flex-md-nowrap align-items-center pb-2 mb-3 border-bottom"
>
  <h1
    className="h2"
  >
    Dashboard
  </h1>
  <div
    className="btn-toolbar mb-2 mb-md-0"
  >
    <div
      className="btn-group mr-2"
    >
      <button
        className="btn btn-sm btn-outline-secondary"
      >
        <Share2
          className="feather"
          color="currentColor"
          size="24"
        />
        Share
      </button>
      <button
        className="btn btn-sm btn-outline-secondary"
      >
        <ExternalLink
          className="feather"
          color="currentColor"
          size="24"
        />
        Export
      </button>
    </div>
    <button
      className="btn btn-sm btn-outline-secondary dropdown-toggle"
    >
      <Calendar
        className="feather"
        color="currentColor"
        size="24"
      />
      This week
    </button>
  </div>
</div>
`;
